x-pack/docs/en/watcher/trigger/schedule/cron.asciidoc

```diff
@@ -1,174 +1,15 @@
-[role="xpack"]
 [[schedule-cron]]
-==== `cron` schedule
+==== Cron schedule
 
-A <<trigger-schedule,`schedule`>> trigger that enables you to use a
-https://en.wikipedia.org/wiki/Cron[cron] style expression to specify when you
-want the scheduler to start the watch execution. {watcher} uses the cron parser
-from the http://www.quartz-scheduler.org[Quartz Job Scheduler]. For more
-information about writing Quartz cron expressions, see the
-http://www.quartz-scheduler.org/documentation/quartz-2.2.x/tutorials/tutorial-lesson-06.html[Quartz CronTrigger Tutorial].
+A <<trigger-schedule, `schedule`>> trigger that enables you to use a
+<<cron-expressions, cron expression>> to specify when you
+want the scheduler to start the watch execution.
```

Comment:
This isn't a complete sentence. Sounds more like a def.

Maybe reword? I know we do this is some other place though.